staging/src/kubevirt.io/client-go/api/v1/types.go

```diff
@@ -1172,11 +1172,11 @@ type KubeVirtSpec struct {
 // +k8s:openapi-gen=true
 type CustomizeComponents struct {
 	// +listType=atomic
-	Patches []Patch `json:"patches,omitempty"`
+	Patches []CustomizeComponentsPatch `json:"patches,omitempty"`
 }
 
 // +k8s:openapi-gen=true
-type Patch struct {
+type CustomizeComponentsPatch struct {
```

Comment:
I am all for this. But we should consider to improve our generating of openapi so this problem doesn't come up in the future.

Reply:
yes. either improve our openapi generation logic, or create some sort of test case that caches when we collide with a Kubernetes api field.

Reply:
I'd prefer the latter.